skate-plugin/src/main/kotlin/com/slack/sgp/intellij/ui/FileChooseDescriptorSkate.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2023 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.slack.sgp.intellij.ui
+
+import com.intellij.openapi.fileChooser.FileChooserDescriptor
+import com.intellij.openapi.vfs.VirtualFile
+
+object FileChooseDescriptorSkate {
```

Comment:
Let's put a doc on this. Also the name is a little confusing, we don't usually put suffixes like this. Let's just call it `FileChoosing`?

Reply:
When you mean doc, do you mean documentation within the skate plugin code?

Reply:
yep! Let's put a doc on this object

```suggestion
/**
 * A meaningful doc explaining what this code is for. Doesn't have to be super detailed, just enough for someone reading this to know what it's for at a glance.
 */
object FileChooseDescriptorSkate {
```

Reply:
Added a doc 👍

Reply:
dd48f90